editor/plugins/ot_features_plugin.cpp

```diff
@@ -156,6 +156,8 @@ void OpenTypeFeaturesAdd::_bind_methods() {
 }
 
 OpenTypeFeaturesAdd::OpenTypeFeaturesAdd() {
+	set_text(RTR("Add Feature..."));
```

Comment:
I know this was just moved around but since this is editor code, it should use `TTR` for translation. `RTR` is only for the runtime (`scene`).